Bottom: da592220eeb434a68bbb6b5ac444be0c433d14f6
Top:    16781ceb9111f967c7ff3b02e2cd0305c5a01478
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-13 01:46:44 +0000

draw.h: Remove unused DrawHoles() prototype


---

diff --git a/src/draw.h b/src/draw.h
index e0688ee..bae0cd8 100644
--- a/src/draw.h
+++ b/src/draw.h
@@ -70,7 +70,6 @@ void DrawLayerGroup (int side, const BoxType *drawn_area);
 void DrawPaste (int side, const BoxType *drawn_area);
 void DrawSilk (HID_DRAW *hid_draw, int side, const BoxType *drawn_area);
 void DrawMask (int side, const BoxType *drawn_area);
-void DrawHoles (bool draw_plated, bool draw_unplated, const BoxType *drawn_area);
 void PrintAssembly (int side, const BoxType *drawn_area);
 
 /* TEMPORARY */
